### PR TITLE
Add depressed font edge

### PR DESCRIPTION
**Pull request by kalvinhuangnbc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ KewtDOM
 |---|---|---|
 | `font` | `'monospace'` | `'monospace'`, `'sans-serif'`, `'serif'` |
 | `fontSize` | `'3'` | `'1'`-`'6'` |
-| `fontEdge` | `none` | `'none'`, `'uniform'`, `'raised'`, `'shadow'`|
+| `fontEdge` | `none` | `'none'`, `'uniform'`, `'raised'`, `'shadow'`, `'depressed'`|
```
Comment (flipactual) [inline README.md]: If you move this change over to `/docs/settings.md` and run `npm run readme`, `README.md` will be generated with the update. It's a little silly at this point, but has helped with organization.

**Comment by flipactual** [comment]
lgtm – just that one small nit and we're gtg

File: README.md (hunk 1 above)
Comment (kalvinhuangnbc) [inline README.md]: oh thats so cool! just made the change

**Comment by flipactual** [comment]
👍

**Comment by flipactual** [comment]
@kalvinhuangnbc v0.2.0 is up [on npm](https://www.npmjs.com/package/kewt)!

**Pull request merged**